### PR TITLE
edits to JOA checklist

### PR DESCRIPTION
**Pull request by wslack**
The length of the checklist makes me want to move some of the content to https://smeqa.usds.gov/hiring-phases/job-announcement/creating-job-announcement/ - thoughts?

### CODE REVIEW

File: pages/phases/job-announcement/index.html [hunk 1]
```diff
@@ -62,19 +62,38 @@ <h4 class="chp-highlight__title">
...
-<h2>Final checklist before JOA Posting</h2>
+<h2>Planning checklist before JOA Posting</h2>
 
 <ul>
-<li>If using USA Staffing, have you set up the manual assessments in order to import the SMEQA assessment results into Staffing? <b>Not doing this will cause a large amount of manual work.</b></li>
-<li>Do you have a final private copy of the competencies and proficiencies that SMEs have approved with language matching the JOA?</li>
+  <li>If using USA Staffing, have you set up the manual assessments in order to import the SMEQA assessment results into Staffing? <b>Not doing this will cause a large amount of manual work.</b></li>
+  <li>Have you determined with HR and hiring managers what assessment materials will be available in the selection process (and explained that on the JOA?)</li>
+  <li>Do you have a final private copy of the competencies and proficiencies that SMEs have approved with language matching the JOA?</li>
+  <li>Is your HR specialist in office and ready to process applicants any day the application may close?</li>
```
Comment (amy-usds) [inline pages/phases/job-announcement/index.html:71]:
```suggestion
  <li>Is your HR specialist in the office and ready to process applicants on any day the application may close?</li>
```

File: pages/phases/job-announcement/index.html [hunk 2]
```diff
@@ -62,19 +62,38 @@ <h4 class="chp-highlight__title">
...
-<li>Is your HR speicialist in office and ready to process applicants any day the application may close?</li>
+<li>Did you consider planning a conference call or calls to inform possible applicants about the application process, and putting the dial in information in the JOA?</li>
 <li>Is your announcement clear about the locations of each posting and if you can hire people remotely? (candidates may assume positions are remote-friendly)</li>
+<li>Do you have the elements on your announcement so that you can make successful offers? For example:</li>
+  <ul>
+      <li>Relocation incentives</li>
+      <li>Recruitment incentives/superior quals statements (these are not the same)</li>
+      <li>Promotion potential
+      <li>Position sensitivity/clearance information</li>
+      <li>Relocation incentives</li>
```
Comment (amy-usds) [inline pages/phases/job-announcement/index.html:91]:
```This is a duplicate list item, remove

File: pages/phases/job-announcement/index.html [hunk 3]
```diff
@@ -62,19 +62,38 @@ <h4 class="chp-highlight__title">
...
+      <li>Drug test requirements</li>
+      <li>Telework eligiblity</li>
```
Comment (amy-usds) [inline pages/phases/job-announcement/index.html:93]:
```suggestion
      <li>Telework eligibility</li>
      <li>Eligibility for 100% remote work, or work at satellite office locations</li>
```

**Review by amy-usds — CHANGES_REQUESTED** [review]
A few inline changes included here (hopefully done right). Please review and then you're approved to merge.

File: pages/phases/job-announcement/create-job-announcement.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ intro: When the content of your well-written job announcement is complete, creat
 
 Agencies using the SME-QA process can use a special, plain-language USAJOBS job announcement template created for this assessment strategy. The template pre-populates many of the Job Opportunity Announcement posting requirements, including the required Legal and Regulatory Guidance section. To create your posting, complete only these sections.
 
-While you'll continue to use the occupational questionnaire to gather general biographical information about the applicant, you won't use the self-rated assessment portion of the questionnaire. Instead, the structured assessment interviews you've documented in your assessment strategy serve as the examination.
+While you'll continue to use the occupational questionnaire to gather general biographical information about the applicant, you won't use the self-rated assessment portion of the questionnaire. Instead, the assessment you've documented in your assessment strategy serve as the examination.
```
Comment (CarleneW) [inline pages/phases/job-announcement/create-job-announcement.md:18]: dd
```suggestion
While you'll continue to use the occupational questionnaire to gather general biographical information about the applicant, you won't use the self-rated assessment portion of the questionnaire. Instead, the structured interviews you've documented in your assessment strategy serve as the examination.
```

Comment (wslack) [reply pages/phases/job-announcement/create-job-announcement.md:18]: rejecting after convo

File: pages/phases/job-announcement/create-job-announcement.md [hunk 5]
```diff
@@ -13,9 +13,9 @@ intro: When the content of your well-written job announcement is complete, creat
...
+Agencies using the SME-QA process can use a special, plain-language USAJOBS job opportunity announcement (JOA) template created for this assessment strategy. The template pre-populates many of the JOA posting requirements, including the required Legal and Regulatory Guidance section. To create your JOA, simply complete the unpopulated sections, as necessary.
 
-While you'll continue to use the occupational questionnaire to gather general biographical information about the applicant, you won't use the self-rated assessment portion of the questionnaire. Instead, the structured assessment interviews you've documented in your assessment strategy serve as the examination.
+While you'll continue to use the occupational questionnaire to gather general biographical information about the applicant, you won't use the self-rated assessment portion of the questionnaire. Instead, the assessment you've documented in your assessment strategy serve as the examination.
```
Comment (wslack) [inline pages/phases/job-announcement/create-job-announcement.md:18]:
```suggestion
While you'll continue to use the occupational questionnaire to gather general biographical information about the applicant, you won't use the self-rated assessment portion of the questionnaire. Instead, the assessment or assessments you've documented serve as the examination.
```

File: pages/phases/job-announcement/index.html (hunk 2 above)
Comment (wslack) [reply pages/phases/job-announcement/index.html:91]: I cna't tell what this is saying?

File: pages/phases/job-announcement/index.html [hunk 6]
```diff
@@ -62,19 +62,39 @@ <h4 class="chp-highlight__title">
...
-<h2>Final checklist before JOA Posting</h2>
+<h2>Planning checklist before JOA Posting</h2>
 
 <ul>
-<li>If using USA Staffing, have you set up the manual assessments in order to import the SMEQA assessment results into Staffing? <b>Not doing this will cause a large amount of manual work.</b></li>
-<li>Do you have a final private copy of the competencies and proficiencies that SMEs have approved with language matching the JOA?</li>
+  <li>If using USA Staffing, have you set up the manual assessments in order to import the SMEQA assessment results into Staffing? <b>Not doing this will cause a large amount of manual work.</b></li>
+  <li>Have you determined with HR and hiring managers what assessment materials (questions asked, applicant responses, and SME analysis) will be available in the selection process (and explained that on the JOA?). We reccomend sharing all materials except for SME scoring.</li>
+  <li>Do you have a final private copy of the competencies and proficiencies that SMEs have approved with language matching the JOA?</li>
+  <li>Is your HR specialist in the office and ready to process applicants on any day the application may close?</li>
```
Comment (wslack) [inline pages/phases/job-announcement/index.html:71]:
```suggestion
  <li>Will your HR specialist be available and ready to process applicants any day that the announcement may close?</li>
```

File: pages/phases/job-announcement/index.html (hunk 2 above)
Comment (wslack) [reply pages/phases/job-announcement/index.html:91]:
```suggestion
```

Comment (amy-usds) [reply pages/phases/job-announcement/index.html]: Yes, sorry, somehow I botched the removal of a duplicate.

**Comment by wslack** [comment]
@CarleneW all of your edits to the JOAa pge look good - is this reaady for merge? It looked like you weren't quite done, but we can merge now and put the rest in later if that works!

File: pages/phases/job-announcement/create-job-announcement.md [hunk 7]
```diff
@@ -26,26 +26,26 @@ Source: 5 U.S.C. § 5105 Standards for classification of positions
...
 
-Include only the competencies SMEs developed during job analysis as the qualifications for your position. You should not be cutting and pasting the OPM Qualification Standards nor from the <a href="{{ site.baseurl }}/hiring-phases/getting-started/identifying-appropriate-positions/">Position Description</a>. As per OPM guidance, you are required to do the work of job analysis to define the specialized experience required, which in our pilots was in the form of competencies and proficiency levels.
+Include only the competencies SMEs developed during job analysis as the qualifications for your position. You should not cut and paste the OPM Qualification Standards nor from the Position Description. In accordance with Federal regulations and OPM guidance, you must conduct a job analysis to define the specialized experience required, which our pilots present in the form of competencies and proficiency levels.
```
Comment (CarleneW) [inline pages/phases/job-announcement/create-job-announcement.md:48]: here, i think it is important to let readers (especially those from HR) know that SMEs are not doing this work alone.  instead, it is SMEs in tandem with HR developing competencies from job analysis.  HR needs to[explicitly] "see themselves" in the process.
```suggestion
Include only the competencies developed with SMEs during job analysis as the qualifications for your position. You should not cut and paste the OPM Qualification Standards nor from the Position Description. In accordance with Federal regulations and OPM guidance, you must conduct a job analysis to define the specialized experience required, which our pilots present in the form of competencies and proficiency levels.
```

**Review by CarleneW — APPROVED** [review]

**Pull request merged**